src/ray/gcs/gcs_server/gcs_server_main.cc

```diff
@@ -38,17 +41,40 @@ DEFINE_string(session_name,
               "session_name: The session name (ClusterID) of the cluster.");
 DEFINE_string(ray_commit, "", "The commit hash of Ray.");
 
+namespace {
+// GCS server output filename.
+constexpr std::string_view kGcsServerLog = "gcs_server.out";
+}  // namespace
+
 int main(int argc, char *argv[]) {
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
+
+  // Backward compatibility notes:
+  // Due to historical reason, GCS server flushes all logging and stdout/stderr to a
```

Comment:
I don't think we have a real "historical reason" to not do rotations. We can choose that as default, but I will just say `// By default, GCS server flushes all logging and stdout/stderr to a single filecalled `gcs_server.out`, without log rotations.` then suggest to use RAY_log_rotation_size to change this.

Reply:
> I don't think we have a real "historical reason" to not do rotations.

When I say "historical reason", I mean the existing implementation, the fact that log rotation is not enabled.

Reply:
Sounds good! Documentation updated.